### PR TITLE
Add error code for internalError

### PR DESCRIPTION
**Pull request by aljones15**
This adds the error `internalError` with a suggested status code of 500.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1127,8 +1127,10 @@ <h2>Example</h2>
 
 <section id="errors">
 	<h1>Errors</h1>
-	<p class="issue">Do we need to define a list of error conditions, codes, etc.</p>
-
+	<h2>internalError</h2>
+    <p>When an unexpected error occurs during did resolution or dereferencing 
+    the resolution or dereferencing meta data error should be <i>internalError</i> and status code 500 
+    should be returned.</p>
```
Comment (kdenhartog) [inline index.html:1133]:
```suggestion
    should be returned when an HTTP(s) binding is in use.</p>
```

**Review by kdenhartog — APPROVED** [review]
One minor nit suggested in order to clarify that the suggestion is for that particular binding. It may make sense for bindings to define these aspects rather than the error section, but for now this is good enough in my opinion.

**Comment by peacekeeper** [comment]
I agree with @kdenhartog that since DID Resolution doesn't necessarily use HTTP, HTTP status codes should be defined in the section about the HTTP(S) Binding:
https://w3c-ccg.github.io/did-resolution/#bindings-https

E.g. that section says if the error is "notFound", then return HTTP status code 404.

But I also agree we could merge this now and then improve later.

**Review by peacekeeper — APPROVED** [review]

**Comment by aljones15** [comment]
@peacekeeper @kdenhartog I can't merge this PR, but I did take the suggestion. So let me know if this is good to go and thanks for the feedback.

**Comment by peacekeeper** [comment]
Looks good, thanks.

File: index.html [hunk 2]
```diff
@@ -1127,8 +1127,10 @@ <h2>Example</h2>
 
 <section id="errors">
 	<h1>Errors</h1>
-	<p class="issue">Do we need to define a list of error conditions, codes, etc.</p>
-
+	<h2>internalError</h2>
+    <p>When an unexpected error occurs during did resolution or dereferencing 
+    the resolution or dereferencing meta data error should be <i>internalError</i> and status code 500
```
Comment (TallTed) [inline index.html:1132]: Got merged too quick...  Needs 2 commas, as shown --
```suggestion
    <p>When an unexpected error occurs during did resolution or dereferencing, 
    the resolution or dereferencing meta data error should be <i>internalError</i>, and status code 500 
```

Comment (peacekeeper) [reply index.html:1132]: Sorry I should have waited longer. I now applied your suggestion in https://github.com/w3c-ccg/did-resolution/commit/746b326570895fc6ce014a659e5d9d55ba9efec0.

**Pull request merged**